packages/common/src/server.ts

```diff
@@ -205,58 +184,13 @@ export class YogaServer<
             }
         : logger
 
-    const maskedErrors = options?.maskedErrors ?? true
-
     this.getEnveloped = envelop({
       plugins: [
         // Use the schema provided by the user
         enableIf(schema != null, useSchema(schema!)),
-        // Performance things
-        enableIf(options?.parserCache !== false, () =>
-          useParserCache(
-            typeof options?.parserCache === 'object'
-              ? options?.parserCache
-              : undefined,
-          ),
-        ),
-        enableIf(options?.validationCache !== false, () =>
-          useValidationCache({
-            cache:
-              typeof options?.validationCache === 'object'
-                ? options?.validationCache
-                : undefined,
-          }),
-        ),
-        // Log events - useful for debugging purposes
-        enableIf(
-          logger !== false,
-          useLogger({
-            logFn: (eventName, events) => {
-              this.logger.debug(eventName)
-              switch (eventName) {
-                case 'execute-start':
-                  const {
-                    query,
-                    variables,
-                    operationName,
-                  }: YogaInitialContext = events.args.contextValue
-                  this.logger.debug(query, 'query')
-                  this.logger.debug(operationName, 'headers')
-                  this.logger.debug(variables, 'variables')
-                  break
-                case 'execute-end':
-                  this.logger.debug(events.result, 'response')
-                  break
-              }
-            },
-          }),
-        ),
-        enableIf(
-          !!maskedErrors,
-          useMaskedErrors(
-            typeof maskedErrors === 'object' ? maskedErrors : undefined,
-          ),
-        ),
+        ...(options?.plugins?.length === undefined
+          ? getDefaultPlugins({ logging: this.logger })
+          : options.plugins),
```

Comment:
Shouldn't this be

```suggestion
          : [...getDefaultPlugins({ logging: this.logger }), ...options.plugins ],
```

As a user I am going to provide plugins, but I still expect parser cache and validation cache to be applied. I did not opt out of those.

For the people that want to own this themselves they have to supply `options.parserCache = false`, right?